ui/components/component-library/select-wrapper/select-wrapper.test.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import { SelectButton } from '../select-button';
+import { SelectWrapper } from '.';
+
+describe('SelectWrapper', () => {
+  it('should render the SelectWrapper without crashing', () => {
+    const { container } = render(
+      <SelectWrapper
+        isOpen={true}
+        triggerComponent={<button>Test Button</button>}
+      >
+        <div>Test</div>
+      </SelectWrapper>,
+    );
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should render the SelectWrapper with additional className and SelectWrapper popover has additional className', () => {
+    const { getByTestId } = render(
+      <SelectWrapper
+        className="mm-select-wrapper mm-test"
+        data-testid="classname"
+        popoverProps={{
+          'data-testid': 'popover-classname',
+          className: 'mm-select-wrapper__popover mm-test',
+        }}
+        triggerComponent={<SelectButton>Test</SelectButton>}
+        isOpen={true}
+      >
+        <div>Test</div>
+      </SelectWrapper>,
+    );
+    expect(getByTestId('classname')).toHaveClass('mm-select-wrapper mm-test');
+    expect(getByTestId('popover-classname')).toHaveClass(
+      'mm-select-wrapper__popover mm-test',
+    );
+  });
+});
```

Comment:
Improve the testing coverage 

<img width="1511" alt="Screenshot 2023-10-17 at 5 11 12 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/05883834-53a9-4c54-bfcb-507e1113a296">